liquigraph-spring-boot-starter/pom.xml

```diff
@@ -84,7 +84,7 @@
             <optional>true</optional>
         </dependency>
         <dependency>
-           <groupId>org.hibernate</groupId>
+           <groupId>org.hibernate.validator</groupId>
```

Comment:
This tag is only indented by 3 spaces. Would you mind adding the fourth?

Reply:
I usually catch those. The indentation is fixed.